aips/aip-43.md

```diff
@@ -39,6 +39,20 @@ AIP 36 removed one serialization point. We can remove the rest with:
 - using Aggregators for total_supply and current_supply counters in the collection.move
 - parallelize event creation, using module events that remove sequence number altogether 
 
+In addition, we will add new API to mint tokens:
+```
+  public fun create_with_index_in_name(
+        creator: &signer,
+        collection_name: String,
+        description: String,
+        name_with_index_preffix: String,
+        name_with_index_suffix: String,
+        royalty: Option<Royalty>,
+        uri: String,
+    ): ConstructorRef {
+```
+which allows minting of the token, where index is part of the name, while still allowing it to happen concurrently.
```

Comment:
Can we clarify whether this is a different name field from the canonical name field every token has (including those not numbered)?

Reply:
you'll see from the other PR, but I am deprecating old name field, introducing new one to be canonical - and this method will set that one